config/routes.yaml

```diff
@@ -52,6 +52,10 @@ register_welcome:
     path:     /register/welcome
     controller: App\Controller\RegistrationController::welcomeAction
 
+register_voucher:
+    path:     /register/voucher/{voucher}
```

Comment:
Why not only `/register/{voucher}`?

Reply:
I would prefer `/register/{voucher}` myself. I chose `register/voucher/{voucher}` to prevent possible colisions with `/register/welcome` and `/register/recovery_token`. Collisions are impossible at the moment as vouchers always have a lenght of six characters, but that might change in the future. I woul be fine with changing the route back to `/register/{voucher}` though.

Reply:
I would prefer that route. So it is also the last route with `/register/*`, even if their is a voucher called welcome they never can use the invite link not the welcome page is not working.

Reply:
Ok, changed route to `/register/{voucher}`